rackcity/models/pdu_port.py

```diff
@@ -24,9 +25,29 @@ class PDUPort(models.Model):
         ],
     )
 
+    class Meta:
+        abstract = True
+
+
+class PDUPort(AbstractPDUPort):
     class Meta:
         constraints = [
             models.UniqueConstraint(
                 fields=['rack', 'left_right', 'port_number'],
                 name='unique port per pdu'),
         ]
+
+
+class PDUPortCP(AbstractPDUPort):
```

Comment:
I think I'm more fine with the port CPs being in the same file bc they're smaller, they don't have a decommissioned counterpart, and they don't really have a ton of logic

Reply:
yeah, I thought it was confusing to have the Abstract class and one of its subclasses in one file, and another one of its subclasses in a diff file. So either theyre all in separate files or theyre all in 1 file. I thought for sake of simplicity I'd combine them all into one file so we dont have a ton of files.